LandLord-latest/src/main/java/biz/princeps/landlord/manager/UtilsManager.java

```diff
@@ -27,14 +20,16 @@ public void sendBasecomponent(Player player, BaseComponent[] message) {
 
     @Override
     public void sendFakeBlockPacket(Player p, Location loc, Material mat) {
-        PacketContainer fakeblock = new PacketContainer(PacketType.Play.Server.BLOCK_CHANGE);
+        // replaced with latest version
+        p.sendBlockChange(loc, mat.createBlockData());
+        /*PacketContainer fakeblock = new PacketContainer(PacketType.Play.Server.BLOCK_CHANGE);
         fakeblock.getBlockPositionModifier().write(0, new BlockPosition(
                 loc.getBlockX(), loc.getBlockY(), loc.getBlockZ()));
         fakeblock.getBlockData().write(0, WrappedBlockData.createData(mat));
         try {
             ProtocolLibrary.getProtocolManager().sendServerPacket(p, fakeblock);
         } catch (InvocationTargetException e) {
             throw new RuntimeException("Cannot send packet " + fakeblock + "! Update ProtocolLib and report back if the error persists!", e);
-        }
+        }*/
```

Comment:
dead code.

Reply:
Left for the upcoming generation to wonder why it is there.